engine/reference/builder.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Dockerfile reference
+description: "Dockerfiles use a simple DSL which allows you to automate the steps you would normally manually take to create an image."
+keywords: build, dockerfile, reference
+toc_max: 3
+redirect_from:
+- /reference/builder/
+---
+
+{% github_content site.buildkit.repo site.buildkit.ref site.buildkit.default_branch "frontend/dockerfile/docs/reference.md" 2 -1 %}
```

Comment:
```suggestion
{% github_content site.buildkit.repo site.buildkit.ref site.buildkit.default_branch "frontend/dockerfile/docs/reference.md" 1 -1 %}
```
I think the whitespace in front will get dropped by jekyll? Then this just becomes "skip the first line" which seems clearer to me.

Reply:
Should be ok as there is an extra line after the frontmatter section